File: robocode2-bot-api-java/src/main/java/net/robocode2/GameType.java
package net.robocode2;

/** Predefined (known) game types. */
public final class GameType {
  private GameType() {}

  /** Melee with every bot against every other bot */
  public final static String MELEE = "melee";

  /** One versus one (1-vs-1) */
  public final static String ONE_VS_ONE ="1v1";

  /** Twin dual with two team both containing two bots */
  public final static String TWIN_DUAL = "twin dual";
}

File: robocode-tankroyale-bot-api-csharp/GameType.cs
namespace Robocode.TankRoyale
{
  /// <summary>
  /// Predefined game types.
  /// </summary>
  public sealed class GameType
  {
    // Hide constuctor.
    private GameType() { }

    /// <summary>Melee battle with every bot against every other bot.</summary>
    public const string Melee = "melee";

    /// <summary>One versus one (1-vs-1) battle.</summary>
    public const string OneVsOne = "1v1";
  }
}